modules/tide_content_collection/css/content-collection-ui-widget.css

```diff
@@ -0,0 +1,10 @@
+div[class*=tabs-advanced-display-sort-criteria]{
```

Comment:
@joshua-salsadigital please ensure that the CSS selector only *targets* the custom widget

Reply:
Added a selector to target the parent custom widget.